pkg/consts/liqonet.go

```diff
@@ -9,4 +9,8 @@ const (
 	DefaultCIDRValue = "None"
 	// TepReady is the ready state of TunnelEndpoint resource.
 	TepReady = "Ready"
+	// IpamStorageResourceLabelKey constant is used for label of resource.
```

Comment:
```suggestion
	// IpamStorageResourceLabelKey is the constant representing the key of the label assigned to all IpamStorage resources.
```

Trying to rephrase, but I am not sure whether I understood correctly (the same for the one below).